pullbackup.py

```diff
@@ -4,9 +4,16 @@
 import datetime
 
 while True:
-    run_command = f'rsync -rv --exclude --remove-source-files "*.new" {local.PULL_BACKUP_FROM} {local.PULL_BACKUP_TO}'
-    print(f'\nNow running rsync at time {datetime.datetime.now()}\nCOMMAND: {run_command}')
-    p = subprocess.run(run_command, shell=True, capture_output=True)
-    print(f'{p.stderr.decode("utf-8")}\n{p.stdout.decode("utf-8")}')
-    print('rsync done!')
-    time.sleep(local.DB_SLEEP_DELAY)
+  # this deletes the files that were backed up, but not the empty directories!
+  run_command = 'rsync -rva --exclude "*.new" --remove-source-files %s %s' % (local.PULL_BACKUP_FROM, local.PULL_BACKUP_TO)
```

Comment:
Looks good but why not use the f-string syntax?

Reply:
Great question @jmccand!

The problem is, on the box where I'm running `pullBackups.py`, I have an old version of Python3 that doesn't have f-strings :(  And I haven't managed to upgrade it yet ...

Reply:
Gotcha. We'll stick with the % based formatting.